helm/aws-load-balancer-controller/templates/webhook.yaml

```diff
@@ -56,7 +56,7 @@ webhooks:
     resources:
     - pods
   sideEffects: None
-{{- if .Values.enableServiceMutatorWebhook }}
+{{- if and .Values.enableServiceMutatorWebhook .Values.serviceMutatorWebhook.enabled }}
```

Comment:
I don't think it's necessary to have `serviceMutatorWebhook.enabled`, probably just use `enableServiceMutatorWebhook`. it needs to be true if someone wants to config service mutator webhook.